### PR TITLE
kr rc co branding

### PR DESCRIPTION
**Pull request by bwgartner**
Based upon the logo (docinfo) changes, per html/pdf output formats, the PR has updated all the k(ubernetes) r(eference) RC documents to include co-branding logos for the respective IHV partners. It also provides some templates (courtesy of Terry) and the main partner logs in the common space.

After review and acceptance, this content can be merged and documents re-published.

### CODE REVIEW

**Review by tomschr — CHANGES_REQUESTED** [review]
@bwgartner Looks good. :+1: Some smaller comments (you are probably already aware):

* There is a conflict, probably from the changes that Frank pushed. You maybe need to rebase your branch.
* To combine the SUSE logo with another partner logo is clever. :wink: 
* Maybe separate the templates from the real logos? You can just create a `templates` folder. This would prevent daps from trying to include or process the template images. If they aren't included anywhere, why process them? IMHO, they aren't needed during building a book.

Hope it helps.

File: kubernetes/reference/adoc/SA_vars.adoc [hunk 1]
```diff
@@ -83,16 +84,42 @@ endif::RA,RC,RI,GS[]
...
+ifdef::IHV-IBM[]
+:subtitle: {subtitle} {vn_IBM} (R)
+:logo: {companyName}-{an_IBM}.svg
+endif::IHV-IBM[]
+ifdef::IHV-LNVGY[]
+:subtitle: {subtitle} {vn_LNVGY} (R)]
+:logo: {companyName}-{vn_LNVGY}.svg
+endif::IHV-LNVGY[]
+ifdef::IHV-SMCi[]
+:subtitle: {subtitle} {vn_SMCi} (R)]
```
[the remark this answers is not in the document]
Comment (bwgartner) [reply kubernetes/reference/adoc/SA_vars.adoc]: Good catch, apologies for missing the cleanup, but fixed that via [09c8a3c](https://github.com/SUSE/technical-reference-documentation/pull/72/commits/09c8a3cce2e82bcaccbc9b1209304266f04e72f1)

**Comment by tomschr** [comment]
> > * There is a conflict, probably from the changes that Frank pushed. You maybe need to rebase your branch.
> 
> Curious what might need to be done in this space

The conflict is due to the `fileref` attribute. You have this line.

```xml
<imagedata fileref="{logo}" width="5em" align="center" valign="bottom"/>
```

whereas in main still has the reference to `suse.svg`:

```xml
<imagedata fileref="suse.svg" width="5em" align="center" valign="bottom"/>
```

As the other attributes are correct, you just need to accept your changes to resolve the conflict.

**Comment by bwgartner** [comment]
> > > * There is a conflict, probably from the changes that Frank pushed. You maybe need to rebase your branch.
> > 
> > 
> > Curious what might need to be done in this space
> 
> The conflict is due to the `fileref` attribute. You have this line.
> 
> ```
> <imagedata fileref="{logo}" width="5em" align="center" valign="bottom"/>
> ```
> 
> whereas main still has the reference to `suse.svg`:
> 
> ```
> <imagedata fileref="suse.svg" width="5em" align="center" valign="bottom"/>

Some (3 docinfo files, GS, and a couple of generic templates) still refer to suse.svg, however the kubernetes reference RI/RC in this PR leverage the respective variable (from it's adoc/SA_vars.adoc) for the right :logo: value (default to suse.svg, unless an RC that co-brands with a partner) to utilize.
 
> ```
> 
> As the other attributes are correct, you just need to accept your changes to resolve the conflict.

**Review by chabowski — COMMENTED** [review]
Thanks! After the conflicts are solved and changes that Tom requested are implemented, I will merge the PR.

**Comment by tlssuse** [comment]
With the latest changes, this looks good to me.

**Comment by chabowski** [comment]
@fsundermeyer Hi Frank, there is still a conflict showing in the kubernetes docinfo file - would you mind having a look? Merging is not possible yet.  Would be great if we could have fixed the issue. Thank you very much!

**Comment by tlssuse** [comment]
It appears that the remaining conflict was addressed verbally by @bwgartner in [this comment](https://github.com/SUSE/technical-reference-documentation/pull/72#issuecomment-1522452091).  
If I read it correctly, Bryan wants to retain use of a variable for specifying the cover logo; that is,
`	    <imagedata fileref="{logo}" width="5em" align="center" valign="bottom"/>
    </imageobject>
    <imageobject role="html">
	    <imagedata fileref="{logo}" width="152px" align="center" valign="bottom"/>`

**Comment by chabowski** [comment]
Hi @tlssuse - yes I read and understood the comment :-) . But I had a look at the files, and I do not know how to resolve the existing conflict - without resolution, the PR cannot be merged. Hoping that @fsundermeyer knows how to address that logo/image issue.

**Pull request merged**